redhat-appstudio/service-provider-integration-operator
# update to kcp 0.8.2 and fix reconcile issue

### What does this PR do?
fixes reconcile on KCP for manual token upload

### Screenshot/screencast of this PR



### What issues does this PR fix or reference?

https://issues.redhat.com/browse/SVPI-199

### How to test this PR?

image `quay.io/mvala/spi-operator:svpi199-manualUploadFixReconcile`

cps is having some issues with permissions for apibinding. After you create your workspace, you need to apply this ClusterRole + ClusterRoleBinding ❗update username to your own :
```
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRole
metadata:
  name: bind-apiexport-spi
rules:
- apiGroups:
  - apis.kcp.dev
  resourceNames:
  - spi
  resources:
  - apiexports
  verbs:
  - bind
---
apiVersion: rbac.authorization.k8s.io/v1
kind: ClusterRoleBinding
metadata:
  name: bind-apiexport-spi
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: ClusterRole
  name: bind-apiexport-spi
subjects:
- apiGroup: rbac.authorization.k8s.io
  kind: User
  name: rh-sso:mvalarh2
```

1. deploy on kcp
2. create spiaccesstokenbinding
3. check `status.uploadUrl` in both binding and token
